benchmarks/osb/params/no-train-params.json

```diff
@@ -10,6 +10,8 @@
   "target_index_bulk_index_data_set_format": "hdf5",
   "target_index_bulk_index_data_set_path": "<path to data>",
   "target_index_bulk_index_clients": 10,
+  "target_index_max_num_segments": 1,
+  "target_index_force_merge_timeout": 45.0,
```

Comment:
is this value in sec, millisec or what? Please sepecifiy.

Reply:
Ill specify in the readme.